components/google-cloud/containers/uCAIP_MB_SDK/cloud_build.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/tfe-ecosystem-dev/mb_sdk_component', '.']
+images: ['gcr.io/tfe-ecosystem-dev/mb_sdk_component']
```

Comment:
Should this be using another project?

Reply:
changed it to a TBD placeholder for now until we lock down the project.